doc/guides/usage-of-primordials-in-core.md

```diff
@@ -0,0 +1,254 @@
+# Usage of primordials in core
+
+The file `lib/internal/per_context/primordials.js` subclasses and stores the JS
+builtins that come from the VM so that Node.js builtin modules do not need to
+later look these up from the global proxy, which can be mutated by users.
+
+Usage of primordials should be preferred for any new code, but replacing current
+code with primordials should be done with care.
+
+## How to access primordials
```

Comment:
I'd prefer no question mark at the end. "(This is) how to access primordials", rather than "How does one access primordials?" If we want to remove the ambiguity, then perhaps this?:

```suggestion
## Accessing primoridals
```